.travis.yml

```diff
@@ -0,0 +1,34 @@
+dist: trusty
+sudo: required
+
+env:
+  - V=HEAD URL_BASE=http://ci.bazel.io/job/Bazel/JAVA_VERSION=1.8,PLATFORM_NAME=linux-x86_64/lastSuccessfulBuild/artifact/output/ci/ URL="$URL_BASE`wget -qO- $URL_BASE | grep -oP 'bazel-.*?-installer.sh' | uniq`" FLAGS=''
+  - V=0.3.2 URL=https://github.com/bazelbuild/bazel/releases/download/0.3.2/bazel-0.3.2-installer-linux-x86_64.sh FLAGS=''
+  - V=0.3.1 URL=https://github.com/bazelbuild/bazel/releases/download/0.3.1/bazel-0.3.1-installer-linux-x86_64.sh FLAGS=''
+
```

Comment:
Firstly, I'm impressed you got bazel caching to work on Travis. I tried to do it. Couldn't make it work.

Secondly, I would advise caution doing this. Right now TensorFlow is doing `bazel clean --expunge` between runs on Jenkins because there's some bug in Bazel regarding cache invalidation of external repositories. We discovered this because I wrote a change that totally changed the directory structure inside external repositories, among other things, and it cross-contaminated test runs. Damien would know more.

This project is small, so the costs of having a fresh environment each time shouldn't be prohibitive.